### ISSUE TITLE
Enter Path (with Directory Name) >> /usr/local/sbin:/usr/local/bin:/usr/sbin: Traceback (most recent call last):   File "/usr/share/hackingtool/hackingtool.py", line 90, in <module>     with open(fpath, "w") as f:          ^^^^^^^^^^^^^^^^

### ISSUE THREAD

**Issue by kingkali07** [Z4nzu/hackingtool#371]
Enter Path (with Directory Name) >> /usr/local/sbin:/usr/local/bin:/usr/sbin:
Traceback (most recent call last):
  File "/usr/share/hackingtool/hackingtool.py", line 90, in <module>
    with open(fpath, "w") as f:
         ^^^^^^^^^^^^^^^^

**Comment by Tamilanth** [comment]
Please give a detailed log of the error,
Include more informations like how to reproduce this error, the platform you are on etc..

**Comment by cclauss** [comment]
Duplicate of #372

**Comment by cclauss** [comment]
@kingkali07 Can this issue be closed or is there a documentation todo?

**Comment by kingkali07** [comment]
issue not closed brother getting same error

**Comment by kingkali07** [comment]
![Screenshot_2023-07-23_00_10_59](https://github.com/Z4nzu/hackingtool/assets/136045841/2b0ef866-4792-4c2f-8dd3-53db48b22bf1)
 brothers please check this image this is the error

**Comment by kingkali07** [comment]
> Please give a detailed log of the error, Include more informations like how to reproduce this error, the platform you are on etc..
![Screenshot_2023-07-23_00_10_59](https://github.com/Z4nzu/hackingtool/assets/136045841/1ca6cd0f-87b0-4792-85b3-7e1d4ce2c00b)

**Comment by kingkali07** [comment]
> @kingkali07 Can this issue be closed or is there a documentation todo?
![Screenshot_2023-07-23_00_10_59](https://github.com/Z4nzu/hackingtool/assets/136045841/3c15dbd7-08ee-413d-82d3-3a18b53606d8)

**Comment by cclauss** [comment]
What filepath did you enter?
The screens above make it look like an empty string was entered.

**Comment by kingkali07** [comment]
I selected 1 manual  after showing this message can you tell me what can I do to solve this problem

**Comment by Tamilanth** [comment]
@kingkali07 In the next step (after choosing the option of manual), it is asking for the path, where the tools can be installed. 

Looks lile you didn't gave any path as input

**Comment by kingkali07** [comment]
ok thankyou brother can you tell me how to gave a path as input please

**Comment by Tamilanth** [comment]
I suggest you to make it default by selecting 2

**Comment by Z4nzu** [comment]
If you're selecting the Manual path option then provide a path like `/home/$USER/downloads` it could be any folder name in which all the tools will be stored.

**Comment by kingkali07** [comment]
solved thankyou all

**Comment by cclauss** [comment]
`$HOME/downloads` might be better on Ubuntu and Debian where $USER is undefined...

% `docker run --rm -it ubuntu:lunar bash`  # Jump into the current Ubuntu as user `root`
```
root@bd928f405d82:/# whoami
root
root@bd928f405d82:/# echo "/home/$USER/downloads"
/home//downloads  # $USER is undefined e.g. an empty string
root@bd928f405d82:/# echo "$HOME/downloads"
/root/downloads
```
docker run --rm --user ubuntu -it ubuntu:lunar bash  # Jump into the current Ubuntu as user`ubuntu`
```
ubuntu@d9869bc6bbdd:/$ whoami
ubuntu
ubuntu@d9869bc6bbdd:/$ echo /home/$USER/downloads
/home//downloads
ubuntu@d9869bc6bbdd:/$ echo $HOME/downloads
/home/ubuntu/downloads
```
---
% `docker run --rm -it debian:bookworm bash`  # Jump into the current **Debian** as user `root`
```
...
```

**Comment by cclauss** [comment]
@kingkali07 If this issue is solved then please close it and also #372.
